addons/web/static/src/js/fields/relational_fields.js

```diff
@@ -794,6 +796,30 @@ var FieldX2Many = AbstractField.extend({
         return this._super.apply(this, arguments);
     },
 
+    /**
+     * @override
+     * @returns {jQuery}
+     */
+    getFocusableElement: function () {
+       return this.mode === 'edit' && this.$input || this.$el;
```

Comment:
I personally prefer parenthesis in this case: is a && b || c the same as (a && b) || c or a && (b || c) ? I don't know...

Reply:
done